torch/_decomp/decompositions.py

```diff
@@ -656,10 +656,13 @@ def logit_backward(
 
 @register_decomposition(aten.native_dropout)
 @pw_cast_for_opmath
-def native_dropout_decomposition(input: Tensor, p: float, train: Optional[bool]):
-    bool_mask = torch.rand_like(input) < p
-    res = bool_mask * input * float(1.0 / p)
-    return [res, bool_mask]
+def native_dropout(input: Tensor, p: float, train: Optional[bool]):
```

Comment:
should it use `training` instead of `train` for consistency with higher-level regular https://pytorch.org/docs/stable/generated/torch.nn.functional.dropout.html?highlight=dropout#torch.nn.functional.dropout ?

Reply:
In general I’ve been going for consistency with the aten schema. I could buy that for ops we’re unifying with the torch api we should use the torch names though (which doesn’t apply here).